lib/makeWebpackConfig.js

```diff
@@ -37,6 +37,11 @@ module.exports = async (playroomConfig, options) => {
 
   const staticTypes = await getStaticTypes(playroomConfig);
 
+  // Use the favicon from the config if specified, otherwise don't use anything
+  const favicon = playroomConfig.faviconPath
+    ? relativeResolve(playroomConfig.faviconPath)
+    : undefined;
```

Comment:
I've defaulted to `undefined` here because I believe the /images folder is not published to npm, and I'm not sure if you want to do that. However, I'm happy to change this if needed.